daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -776,7 +776,7 @@ func (devices *DeviceSet) poolHasFreeSpace() error {
 
 	minFreeMetadata := (metadataTotal * uint64(devices.minFreeSpacePercent)) / 100
 	if minFreeMetadata < 1 {
-		minFreeData = 1
```

Comment:
I think we can just use this one

On Thursday, March 17, 2016, Vivek Goyal <notifications@github.com> wrote:

> In daemon/graphdriver/devmapper/deviceset.go
> <https://github.com/docker/docker/pull/21274#discussion_r56516653>:
>
> > @@ -776,7 +776,7 @@ func (devices *DeviceSet) poolHasFreeSpace() error {
> >
> >  	minFreeMetadata := (metadataTotal * uint64(devices.minFreeSpacePercent)) / 100
> >  	if minFreeMetadata < 1 {
> > -		minFreeData = 1
>
> Yes, this is a bug. it should have been minFreeMetadata=1. Will generate a
> PR to fix it. Thanks for catching this.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/21274/files/a7d721626a4b3b2a7c510fc1d2d080fe7d64ccc2#r56516653>
>


-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>